src/Api/AppServices.hs

```diff
@@ -39,6 +38,8 @@ import Servant (Handler, err500, err401, err403)
 -- transformers
 import Control.Monad.Trans.Except (ExceptT, runExceptT)
 
+import qualified Tagger.Database as DB
```

Comment:
This is addressed with the formatting the project anyway, so I didn't bother. For the sake of simplicity, I would ask to leave it as such since the other PR is already open.